source/sync/configure/enable-sync.txt

```diff
@@ -124,7 +124,9 @@ Procedure
                  "state": <"enabled" | "disabled">,
                  "client_max_offline_days": <Number>,
                  "is_recovery_mode_disabled": <Boolean>,
-                 "queryable_fields_names": <Array of String Field Names>
+                 "indexed_queryable_fields_names": <Array of String Field Names>,
```

Comment:
Just as a note, it might be a bit confusing that we only allow one indexed_queryable_field but that this is an array (This is to allow us to potentially support more than 1 IQF in the future). 

Would it be possible to add a note here that this array should contain maximum 1 element (here or in the other section describing the config fields)?

Reply:
It's our convention to list only the type in the config object example, but I've got more details in the table that describes each of the fields: https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-31839/sync/configure/sync-settings/#sync-config-object I'll rework this a bit to be even more explicit.

Reply:
Cool, this copy in the table seems explicit enough to me. Thanks!